client/src/components/grants/Details.tsx

```diff
@@ -20,16 +22,52 @@ export default function Details({
   bannerImg,
   logoImg,
   preview,
+  alert,
 }: {
   project?: Metadata | FormInputs | Project;
   updatedAt: string;
   createdAt: string;
   bannerImg: string | Blob;
   logoImg: string | Blob;
   preview?: boolean;
+  alert?: boolean | undefined;
 }) {
+  const [showAlert, setShowAlert] = useState(alert ?? true);
+  const discordLink = (
+    <a className="text-purple-600" href="https://discord.gg/nwYzGuuruJ">
+      Grant Hub Discord
+    </a>
+  );
+
+  const successfullSubmissionBody = (
+    <p className="text-black">
+      Your application will be reviewed by the grant round team and you will
+      receive an email if your project is approved for the grant round. If you
+      have any questions or feedback, feel free to reach us out on the{" "}
+      {discordLink}!
+    </p>
+  );
+
+  // ToDo: make the round name dynamic
+  const successfullSubmissionTitle = (
+    <p className="text-green-800 text-xl">
+      Thank you for applying to Optimism Grant Round!
```

Comment:
Should this come from the round metadata?

Reply:
@gravityblast don't we need to update our state object to handle this updated property `roundName` from round-manager? They added the round name last week to the metadata we just need the address to call it 🤔 looking to update this, can we pair tomorrow and tackle this?

<img width="399" alt="Screen Shot 2022-09-27 at 3 38 05 PM" src="https://user-images.githubusercontent.com/9419140/192619876-7fdfd75b-34d7-43f2-858c-1755db040668.png">

Reply:
Nah, you can leave it - I'll tackle it as part of #386